libs/core/langchain_core/messages/human.py

```diff
@@ -4,11 +4,33 @@
 
 
 class HumanMessage(BaseMessage):
-    """Message from a human."""
+    """Message from a human.
+
+    HumanMessages are messages that are passed in from a human to the model.
+
+    Example:
+
+        .. code-block:: python
+
+            from langchain_core.messages import HumanMessage, SystemMessage
+
+            messages = [
+                SystemMessage(
+                    content="You are a helpful assistant! Your name is Bob."
+                ),
+                HumanMessage(
+                    content="What is your name?"
+                )
+            ]
+
+            # Instantiate a chat model and invoke it with the messages
+            model = ...
+            print(model.invoke(messages))
+    """
 
     example: bool = False
     """Whether this Message is being passed in to the model as part of an example 
-        conversation.
+    conversation.
```

Comment:
@baskaryan do you know if example is actually used / respected in any models?

Reply:
was for some of the original google palm models but afaict doesn't apply anymore